src/main/java/org/openstreetmap/atlas/geography/atlas/items/Relation.java

```diff
@@ -93,6 +94,20 @@ public String configurableString(final String betweenEachMemberAndRelation,
         return builder.toString();
     }
 
+    /**
+     * @return the {@link RelationBean} representation of the Relation
+     */
+    public RelationBean getBean()
+    {
+        final RelationBean bean = new RelationBean();
+        for (final RelationMember member : this)
+        {
+            bean.addItem(member.getEntity().getIdentifier(), member.getRole(),
```

Comment:
Nit: for a large Relation - any performance impact on pulling out the entity just once and saving as a temporary variable, and then getting the identifier and type instead of fetching the entity each time.